### PR TITLE
Improve session termination reason handling WIP

### PR DESCRIPTION
**Pull request by fholzhauser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,10 @@ Example of configuration **RADIUS**:
                 {link_broken, 2},
                 {upf_failure, 9},
                 {remote_failure, 9},
-                {inactivity_timeout, 4},
+                {cp_inactivity_timeout, 4},
```
Comment (vkatsuba) [inline README.md:86]: Please update also `METRICS.md` and add:
```
 * cp_inactivity_timeout
 * up_inactivity_timeout
 * aaa_asr
 * aaa_error
 * aaa_req_timeout
 * aaa_conn_error
 * aaa_rate_limit
 * aaa_ocs_hold_end
```
in end of `The label type is the Termination Causes types. For Termination causes the following types exist:` section

**Comment by RoadRunnr** [comment]
@fholzhauser is this PR still valid (and needs to be rebased) or should it be closed?

**Comment by fholzhauser** [comment]
I'll rebase it and fix it up for review

**Comment by vkatsuba** [comment]
Continued implementation after context refactoring in https://travis-ci.com/travelping/ergw/builds/204961968?utm_source=slack&utm_medium=notification

**Pull request closed without merging**